scenarios/kill_wf_attempt/README.md

```diff
@@ -5,9 +5,19 @@
 ## What is the purpose of this scenario?
 To kill the attempt automatically if the attempt violates sla.
 
-# Prerequisity
+## Prerequisity
 - python custom scripting is enabled on your account
 
+## Regarding baseurl
+"baseurl" in kill_wf_attempt.dig is up to your account's site.
+You have to change baseurl if your account's site is EU or Tokyo.
+
+|Sites|Endpoints|
```

Comment:
Might be better to refer [the doc](https://support.treasuredata.com/hc/en-us/articles/360001474288-Sites-and-Endpoints#Endpoints)